client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -429,7 +431,28 @@ export class DomainDetailsForm extends PureComponent {
 	focusFirstError() {
 		const firstErrorName = kebabCase( head( formState.getInvalidFields( this.state.form ) ).name );
 		const firstErrorRef = this.inputRefs[ firstErrorName ] || this.refs[ firstErrorName ];
-		firstErrorRef.focus();
+
+		try {
+			firstErrorRef.focus();
+		} catch ( err ) {
+			const noticeMessage = this.props.translate(
+				'There was a problem validating your contact details: "%(firstErrorName)s" required. ' +
```

Comment:
Could we make `%(firstErrorName)s` a component as well? `{{firstErrorName/}}` and the component be something like `<strong>{firstErrorName}</strong>`